doc/content/gateways/thethingsindoorgateway/_index.md

```diff
@@ -0,0 +1,112 @@
+
+### Procedure
+
+Go to **Gateways** in the top menu, and click **Claim Gateway** to reach the gateway claiming page.
+
+Fill the **Gateway EUI**, the **Claim authentication code** (the WiFi Password), the **Gateway ID** and choose the appropriate **Frequency Plan**.
+
+Click **Claim gateway** to finish.
+
+{{< figure src="TTIG_Claim.png" alt="{{% ttig %}} Claiming" >}}
+
+If your inputs are correct, a new gateway will be created and you will be redirected to the gateway overview page of your newly created gateway.
+
+## Connecting {{% ttig %}}
+
+### New Gateways
+
+Press the reset button (small button at the back of the gateway next to the USB-C port) for 5 seconds until the LED blinks rapidly GREEN<->RED for a couple of times.
+
+Hold the SETUP (button at the top of the gateway, next to the LED) for 10 seconds until the LED blinks RED rapidly.
+
+The gateway now exposes a WiFi AP whose SSID is **MINIHUB-xxxxxx** where `xxxxxx` is the last 6 digits of the gateway EUI. The password for this network is printed on the back panel of the device under **WiFi PW**.
+
+After connecting to this network go to *192.168.4.1* using a web browser to access the WiFi config page.
+{{< figure src="TTIG_WiFi_Setup.png" alt="{{% ttig %}} WiFi Setup" >}}
+
+Select the WiFi network by clicking the **+** button on the desired network (and enter the password if it's a secure network).
+
+{{< note >}} {{% ttig %}} does not support enterprise WiFi networks. {{</ note >}}
+
+Select the **Save and Reboot** option. 
+
+If your configuration is correct, 
+  * The gateway will blink GREEN for a few seconds while it connects to this network.
+  * Then, it will blink GREEN<->RED for a few seconds while it connects to the server and fetches the necessary configuration.
+  * Please allow 5-10 mins for the gateway to pick up new configuration. 
+
+{{< note >}} If this is the first time your gateway is being powered on/connected to WiFi, it might pickup a new firmware depending on when it was last updated. This is indicted by alternating GREEN/RED blinks of the LED. Please leave the gateway powered on when this happens. {{</ note >}}
+
+### Gateways Connected to Another LNS
+
+If the {{% ttig %}} is currently connected to another LNS, do either of the following
+- Wait up to 24 hrs (this is the default action for gateways that are not physically reachable). {{% ttig %}} connects to the server every 24 hours to check for new configuration.
+- Unplug the power supply, wait for a few seconds and plug it back in. {{% ttig %}} now connects to the server to check for new configuration.
+
+## Troubleshooting
+
+### Common Errors
+
+#### Gateway Not Authorized for Claiming
+
+If you have already registered your TTIG, you have the following options
+
+a. If you would like to retain the Gateway ID that you registered it with, support will be added in a subsequent release to claim the gateway while retaining it's current Gateway ID.
```

Comment:
```suggestion
1. If you would like to retain the Gateway ID that you registered it with, support will be added in a subsequent release to claim the gateway while retaining its current Gateway ID.
```

Reply:
I don't understand this, you mean the support to do this will be added in the next TTS release?

Reply:
Yes that's what it means.